src/server/passport/strategyUtils.js

```diff
@@ -0,0 +1,56 @@
+const responseManager = require('../routes/controllers/utilities/responseManager')
+const Utilities = require('../utilities')
+
+const STRATEGY_NAME = 'LOGIN_STRATEGY'
+const FIELDS_NAMES = {
+	username_field: 'username',
+	password_field: 'password'
+}
+
+function strategyHandlers(req, res, next) {
+	const serializeHandler = (err) => {
+		if (err) return responseManager.createResponse(res, err)
+		
+		next()
+	}
+
+	const loginStrategyHandler = (err, user, info) => {
+		if (err)
+			return responseManager.createResponse(res, err)
+
+		if (user)	
+			req.logIn(user, serializeHandler)
+		else
+			return responseManager.createResponse(res, responseManager.MESSAGES.errors.UNAUTHORIZED)
+	}
+
+	return {
+		serializeHandler,
+		loginStrategyHandler
+	}
+		
+}
+
+function validateUserPassword(username, password, done) {
+	const foundUser = user => {
```

Comment:
`foundUser` doesn't mean anything from function POV. Actually it looks more like object, or boolean value.
Please give it a descriptive name.

Reply:
ok